src/hatti/utils.cljs

```diff
@@ -54,3 +54,8 @@
   "Create an appropriately pluralized string prefix by number."
   [number kind]
   (join " " [number (if (= 1 number) kind (plural kind))]))
+
+(defn in?
```

Comment:
Cool, at some point in the future we need a clojure utils library, we have this in zebra, hatti, and milia